core/src/test/scala/unit/kafka/admin/DeleteTopicTest.scala

```diff
@@ -277,10 +277,10 @@ class DeleteTopicTest extends ZooKeeperTestHarness {
     servers.foreach(_.shutdown())
   }
 
-  private def createTestTopicAndCluster(topic: String): Seq[KafkaServer] = {
+  private def createTestTopicAndCluster(topic: String, deleteTopicEnabled: String = "true"): Seq[KafkaServer] = {
```

Comment:
Sorry did not notice this earlier. This is slightly ugly. Would prefer taking an actual boolean arg and convert that to a string inside.